site/en/docs/webstore/cws-dashboard-distribution/index.md

```diff
@@ -90,13 +90,35 @@ the Chrome Web Store. You can either:
 When you publish to the world, your item will be immediately visible to the region(s) you select,
 and it'll be visible in the store's search results.
 
+## Managing a test version and a production version
+
+You can publish a "BETA" or "TESTING" version to the Chrome Web Store in parallel with a
+production version. It can be listed private, unlisted or public. After addressing issues and bugs based on user
+feedback, you can publish a revised version of your extension to the rest of your users. 
+
+Before you submit a testing
+  version to the Chrome Web Store, follow these steps:
+
+  1.  Add the label "DEVELOPMENT BUILD" or “BETA” at the end of the name of your extension. 
+  2.  Declare the
+  purpose of your extension in the description: "THIS EXTENSION IS FOR BETA TESTING".
+
+{% Aside 'warning' %}
+
+Missing these steps can cause both your extensions to be taken down. For more information,
```

Comment:
Consider: "Make sure to add these labels. If you don't, then both extensions may be taken down."